charts/ocis/values.yaml

```diff
@@ -21,170 +47,397 @@ image:
+      # type: array
+      # items:
+      #   type: string
+      #   required: true
+      # @schema
       # -- img-src directive, see https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy/img-src
       imgSrc:
         - "'self'"
         - "data:"
         - "blob:"
+      # @schema
+      # type: array
+      # items:
+      #   type: string
+      #   required: true
+      # @schema
       # -- manifest-src directive, see https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy/manifest-src
       manifestSrc:
         - "'self'"
+      # @schema
+      # type: array
+      # items:
+      #   type: string
+      #   required: true
+      # @schema
       # -- media-src directive, see https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy/media-src
       mediaSrc:
         - "'self'"
+      # @schema
+      # type: array
+      # items:
+      #   type: string
+      #   required: true
+      # @schema
       # -- object-src directive, see https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy/object-src
       objectSrc:
         - "'self'"
         - "blob:"
+      # @schema
+      # type: array
+      # items:
+      #   type: string
+      #   required: true
+      # @schema
       # -- script-src directive, see https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy/script-src
       scriptSrc:
         - "'self'"
         - "'unsafe-inline'"
+      # @schema
+      # type: array
+      # items:
+      #   type: string
+      #   required: true
+      # @schema
       # -- style-src directive, see https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy/style-src
       styleSrc:
         - "'self'"
         - "'unsafe-inline'"
 
 # Debug settings for the oCIS service
 debug:
+  # @schema
+  # type: boolean
+  # required: true
+  # @schema
   # -- Profiling enables the http://<pod>:<debug-metrics>/debug/pprof endpoint to inspect various Go runtime internals.
   # You can use the endpoint on your machine by forwarding the port, eg: `kubectl port-forward -n ocis pod/proxy-8587dc9d64-fs24l 9205:9205`
   # and then accessing the port on https://localhost:9147/debug/pprof or using the pprof command line tool: `go tool pprof -web http://localhost:9205/debug/pprof/symbol\?seconds\=10`
   profiling: false
 
-# -- Deployment strategy.
+# Deployment strategy.
 deploymentStrategy:
+  # @schema
+  # enum: [RollingUpdate, Recreate]
+  # required: true
+  # @schema
+  # -- deployment strategy type to use. Can be "RollingUpdate" or "Recreate"
   type: RollingUpdate
+  rollingUpdate:
+    # @schema
+    # type: string
+    # required: true
+    # @schema
+    # -- The maximum number of pods that can be scheduled above the desired number of pods.
+    # Value can be an absolute number (ex: 5) or a percentage of desired pods (ex: 10%).
+    maxSurge: 25%
+    # @schema
+    # type: string
```

Comment:
`IntOrString` same as `maxSurge`

Reply:
Good point!

was applied in https://github.com/owncloud/ocis-charts/pull/811/commits/f793e618c47751812259088ef79f2d84849338cf